broker/pom.xml

```diff
@@ -146,6 +146,11 @@
       <artifactId>netty-common</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
:question: This is added only so we can pass it down to the gateway config, right?